src/test/java/seedu/todo/logic/commands/CompleteCommandTest.java

```diff
@@ -46,4 +50,42 @@ public void testMarkIncomplete() throws Exception {
         assertEquals(markedIncomplete, toMarkIncomplete);
         assertFalse(toMarkIncomplete.isCompleted());
     }
+    
+    @Test
+    public void testCompleteAll_withView() throws Exception {
+        model.view(TaskViewFilter.INCOMPLETE);
+        List<ImmutableTask> tasks = model.getObservableList();
+        setParameter("all", "all");
+        execute(true);
+        int count = 0;
+        for (ImmutableTask task : tasks){
+            assertTrue(task.isCompleted());
+            if(task.isCompleted()){
+                count++;
+            }
+        }
+        assertEquals(tasks.size(), count);
+    }
+
+    @Test
+    public void testCompleteAll_withAlreadyComplete() throws Exception {
+        setParameter("all","all");
+        execute(true);
+        int count = 0;
+        for (ImmutableTask task : model.getObservableList()){
+            assertTrue(task.isCompleted());
+            if(task.isCompleted()){
+                count++;
+            }
+        }
+        assertEquals(model.getObservableList().size(), count);
```

Comment:
This assertion serves no purpose - it is logically impossible for this to fail if all previous assertions have passed